jira-doc.py

```diff
@@ -0,0 +1,137 @@
+    inprogress = jira.search_issues(query_string)
+    print ('In progress')
+    file.write('In progress \n')
+    for x in inprogress:
+        # update_date = x.raw['fields']['updated']
+        # update_date = update_date.split('.', 1)[0]
+        # then = datetime.strptime(update_date, "%Y-%m-%dT%H:%M:%S")
+        # if then > datetime.now() - timedelta(days = 7):
+            if x.raw['fields']['assignee']is None:
+                if len(x.fields.customfield_10010) > 1:
+                    opened = 'This task was included in '+ str(len(x.fields.customfield_10010)) + ' sprints.'
+                else:
+                    opened = ''  
+                output = (x.raw['fields']['summary']+'. '+ opened)
+                file.write(output.encode('utf-8') + "\n")
+            else:
+                if len(x.fields.customfield_10010) > 1:
+                    opened = 'This task was included in '+ str(len(x.fields.customfield_10010)) + ' sprints.'
+                else:
+                    opened = ''
+
+                output = (x.raw['fields']['summary'] +' '+ x.raw['fields']['assignee']['displayName']+ '. ' + opened)
+
+                file.write(output.encode('utf-8') + "\n")
+
+    query_string = ('project=JD and status="To do" and Sprint="' + sprint +'"')
+    todo = jira.search_issues(query_string)
+    print ('To do')
+    file.write('To do \n')
+    for x in todo:
+        # update_date = x.raw['fields']['updated']
+        # update_date = update_date.split('.', 1)[0]
+        # then = datetime.strptime(update_date, "%Y-%m-%dT%H:%M:%S")
+        # if then > datetime.now() - timedelta(days = 7):
+            if x.raw['fields']['assignee']is None:
+                if len(x.fields.customfield_10010) > 1:
+                    opened = 'This task was included in '+ str(len(x.fields.customfield_10010)) + ' sprints.'
+                else:
+                    opened = ''
+                output = (x.raw['fields']['summary']+'. '+ opened)
+                
+                file.write(output.encode('utf-8') + "\n")
+            else:
+                if len(x.fields.customfield_10010) > 1:
+                    opened = 'This task was included in '+ str(len(x.fields.customfield_10010)) + ' sprints.'
+                else:
+                    opened = ''
+
+                output = (x.raw['fields']['summary'] +' '+ x.raw['fields']['assignee']['displayName']+ '. ' + opened)
+
+                file.write(output.encode('utf-8') + "\n")
+    file.close()
+
+with open('creds.txt') as f:
+  creds = [x.strip().split(':') for x in f.readlines()]
+for username,password in creds:
+    with open('config.yaml') as l:
+        url = l.readlines()
+    for server in url:
+        options = {
+            'server': server
+            }
+jira = JIRA(options, basic_auth = (username, password))
+check_sprint = jira.search_issues('project=JD and status="In progress"')
+sprint = ''
+status = ''
+empty_list = []
+# print (check_sprint)
+for x in check_sprint:
```

Comment:
Rewrite this as a "get current sprint" function, return the sprint Id, and call the `get_list` function afterwards. It will be easier to understand the flow